src/Symfony/Component/Routing/Loader/Configurator/RoutingConfigurator.php

```diff
@@ -25,7 +27,7 @@ class RoutingConfigurator
     private $path;
     private $file;
 
-    public function __construct(RouteCollection $collection, PhpFileLoader $loader, string $path, string $file)
+    public function __construct(RouteCollection $collection, LoaderInterface $loader, string $path = null, string $file = null)
```

Comment:
@Tobion , this class initially was designed to work within a `FileLoader`, so I made some adjustments, looking at the `RouteCollectionBuilder`. Probably this is the wrong approach, I am ready to learn a better one :)